osu.Game.Rulesets.Mania/UI/Components/ColumnType.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Mania.UI.Components
+{
+    public abstract class ColumnType
+    {
+        public abstract Color4 ColumnColour { get; }
+        public virtual float DefaultColumnWidth => Column.COLUMN_WIDTH;
+
+        public virtual string NoteTexture => "mania-note1";
+        public virtual string HoldTexture => "mania-note1L";
+        public virtual string HoldHeadTexture => "mania-note1H";
+        public virtual string HoldTailTexture => "mania-note1T";
+        public virtual string KeyTexture  => "mania-key1";
+        public virtual string KeyDownTexture => "mania-key1D";
```

Comment:
These values does not point to the actual textures but the name of the textures.

Another name could be better.

Reply:
perhaps `...TextureName`?